Integrations/integration-CiscoUmbrellaInvestigate.yml

```diff
@@ -0,0 +1,226 @@
+commonfields:
+  id: Cisco Umbrella Investigate
+  version: -1
+system: true
+name: Cisco Umbrella Investigate
+display: Cisco Umbrella Investigate
+category: Vulnerability Management
```

Comment:
Im not sure really what the category should be? Is it a Utility? It returns how malicious a domain but it also returns domain information so I didn't know what category it should be in which I would call a Vuln management tool as it ranks the domain for how malicious the domain is. Vuln management products also seem to do that as well.